src/cephadm/cephadm

```diff
@@ -7589,6 +7583,10 @@ def _get_parser():
         type=str,
         default=get_hostname().split('.')[0],
         help='daemon identifer for the exporter')
+    parser_exporter.add_argument(
+        '--binary',
+        type=str,
+        help='path to the cephadm binary')
```

Comment:
What's the use case for this arg? If we're calling the exporter command directly from the command line then the binary to use would just be the one we're calling with, and in a call from the mgr to setup an exporter the only place the binary location is being used is in the CephadmDaemon unit_run function https://github.com/ceph/ceph/blob/4166eca33dcb242f5f60262033737867ec8fde59/src/cephadm/cephadm#L6829-L6838 which we aren't adjusting to use this arg.

Reply:
> What's the use case for this arg? If we're calling the exporter command directly from the command line then the binary to use would just be the one we're calling with.

There is no binary at this point. At least not when calling from cephadm. At least not until we have merged #39619 which depends on this very PR. 

but we can probably avoid adding the --binary command argument indeed.